std/regex/internal/backtracking.d

```diff
@@ -628,6 +654,8 @@ template BacktrackingMatcher(bool CTregex)
                     case IR.LookbehindEnd:
                     case IR.NeglookbehindEnd:
                     case IR.End:
+                        // cleanup stale stack blocks if any
+                        while (prevStack()) {}
```

Comment:
I don't get it - CircleCI keeps naging about this line - space after 'while'. But in the latest commit there IS a space between while and paren.